core/src/main/scala/com/broadinstitute/dsp/checkerDeps.scala

```diff
@@ -41,17 +41,21 @@ object RuntimeCheckerDeps {
                                                               blockerBound,
                                                               RetryPredicates.standardRetryConfig)
     } yield {
-      RuntimeCheckerDeps(appConfig.reportDestinationBucket, computeService, storageService, dataprocService)
+      val checkRunnerDeps = CheckRunnerDeps(appConfig.reportDestinationBucket, storageService)
+      RuntimeCheckerDeps(computeService, dataprocService, checkRunnerDeps)
     }
 }
 
-final case class Runtime(googleProject: GoogleProject, runtimeName: String, cloudService: CloudService) {
-  override def toString: String = s"${googleProject.value},${runtimeName},${cloudService}"
+final case class Runtime(id: Long,
+                         googleProject: GoogleProject,
+                         runtimeName: String,
+                         cloudService: CloudService,
+                         status: String) {
+  override def toString: String = s"$id,${googleProject.value},${runtimeName},${cloudService},$status"
```

Comment:
this is mostly for report files.

when you look at value, it's actually relatively easy to know what each column is...reason why I'm outputting this way is because we can use this as input type for scripts directly..if we do the format you mentioned, it'll be a lot more work to parse that

Reply:
> reason why I'm outputting this way is because we can use this as input type for scripts directly
I see. It may be good to add that as a comment.